lib/root_loader.rb

```diff
@@ -1,6 +1,7 @@
 # frozen_string_literal: true
 
 require 'json'
+require 'logs'
```

Comment:
So what happens if this is missing? Why did we need to add it?

Reply:
It gives an error regarding not being able to find the following class and hence fails. 
```
PrimarySchema = Util::TrackableErrorClass.new(
        msg:  "Primary schema is {0-primary-schema}",
        code: "CONJ00034I"
      )
``` 
found in [logs.rb](https://github.com/cyberark/conjur/blob/f901b847da8cdc10b659d71167cba5f406b1eb77/app/domain/logs.rb#L13)
If you would like I can run the script and copy and paste the logs

Reply:
ah. that's code that i added recently. weird that it doesn't fail the build in jenkins. Thanks for the explanation.